localstack/testing/snapshots/transformer_utility.py

```diff
@@ -165,18 +165,36 @@ def sqs_api():
         """
         return [
             TransformerUtility.key_value("ReceiptHandle"),
-            TransformerUtility.key_value("SenderId"),
+            # account ID
+            TransformerUtility.key_value(
+                "SenderId", value_replacement="111111111111", reference_replacement=False
```

Comment:
@bentsku: is there a specific reason why you decided to replace the senderId with 1s? IMO it can be a bit misleading, as we also have the regex-replacement for the account-id, and on first sight it will look like the regex-replacement is responsible for this replacement. 

Personally, I would prefer if we could use distinct replacements values for transformers.

Reply:
I am going to test it again, but I believe the issue was that locally, the `SenderId` was replaced by 1s, and not in the tests against AWS. I'll check again, could be priorities, I need to check what the response was before transformation.

Reply:
This is now changed and simply use `reference_replacement=False` instead, as it is enough to not mess with ARN anymore